src/Fantomas.Core/CodePrinter.fs

```diff
@@ -3037,7 +3045,8 @@ let genType (t: Type) =
         +> optSingle genIdentListNodeWithDot node.PostIdentifier
         +> genSingleTextNode node.LessThen
         +> addExtraSpace
-        +> col sepComma node.Arguments genType
+        +> leadingExpressionIsMultiline (colGenericTypeParameters node.Arguments) (fun isMultiline ->
```

Comment:
We should add some tests where the `<` and `>` contain trivia.
Cases like:
```fsharp
type X =
    Teq< //
        int
     //
     >
```
won't be covered right now.

Reply:
Also please add a code comment on why we are doing this in the first place.